lib/ProductOpener/DataQualityFood.pm

```diff
@@ -1171,7 +1171,7 @@ sub check_categories ($product_ref) {
 
 	# Check alcohol content
 	if (has_tag($product_ref, "categories", "en:alcoholic-beverages")) {
-		if (!(defined $product_ref->{alcohol_value}) || $product_ref->{alcohol_value} == 0) {
+		if (!(defined $product_ref->{nutriments}->{alcohol_value}) || $product_ref->{nutriments}->{alcohol_value} == 0) {
```

Comment:
We don't need the 2nd -> (I didn't know it worked in fact :) )

```suggestion
		if (!(defined $product_ref->{nutriments}{alcohol_value}) || $product_ref->{nutriments}->{alcohol_value} == 0) {
```

Reply:
see last commit